crypto/fipsmodule/cpucap/internal.h

```diff
@@ -168,6 +168,14 @@ OPENSSL_INLINE int CRYPTO_is_ARMv8_PMULL_capable(void) {
 #endif
 }
 
+OPENSSL_INLINE int CRYPTO_is_ARMv8_GCM_8x_capable(void) {
```

Comment:
see my comment in the `arm_arch.h` file. `GCM_8x` is not a capability of the CPU, the others are all capabilities (`NEON`, `AES`, `PMULL`).

Reply:
Done